plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/machine/MachineStatusHandler.java

```diff
@@ -119,6 +120,12 @@ private void handleMachineRunning(final String machineId, final WorkspaceRuntime
         if (machine == null) {
             return;
         }
-        eventBus.fireEvent(new MachineStateEvent(machine, RUNNING));
+
+        try {
+            eventBus.fireEvent(new MachineStateEvent(machine, RUNNING));
```

Comment:
Firing an event wrapped in try block...
Never saw it before in our codebase.
That's interesting. Can you explain, please, what's the motivation of this